pyfftw/interfaces/_utils.py

```diff
@@ -126,7 +131,4 @@ def _Xfftn(a, s, axes, overwrite_input, planner_effort,
         FFTW_object(input_array=a, output_array=output_array, 
                 normalise_idft=normalise_idft)
     
```

Comment:
Two of the transform types will always clobber the input, so it's not safe to drop the reload after input for those.